### PR TITLE
ENCD-4345-refactor-result-from-set-filter

### PR DESCRIPTION
**Pull request by zoldello**

### CODE REVIEW

**Comment by zoldello** [comment]
@caseylitton I updated the name of this repo.

File: buildout.cfg [hunk 1]
```diff
@@ -53,7 +53,7 @@ webtest = git https://github.com/Pylons/webtest.git
 WSGIProxy2 = git https://github.com/lrowe/WSGIProxy2.git
 zope.sqlalchemy = git https://github.com/zopefoundation/zope.sqlalchemy.git
 pytest-bdd = git https://github.com/lrowe/pytest-bdd.git branch=allow-any-step-order
-snovault = git https://github.com/ENCODE-DCC/snovault.git rev=1.0.21
+snovault = git https://github.com/ENCODE-DCC/snovault.git branch=SNO-69-backend-matrix
```
[the remark this answers is not in the document]
Comment (zoldello) [reply buildout.cfg]: Is there anything need for me to do for this?

File: buildout.cfg [hunk 2]
```diff
@@ -53,7 +53,7 @@ webtest = git https://github.com/Pylons/webtest.git
 WSGIProxy2 = git https://github.com/lrowe/WSGIProxy2.git
 zope.sqlalchemy = git https://github.com/zopefoundation/zope.sqlalchemy.git
 pytest-bdd = git https://github.com/lrowe/pytest-bdd.git branch=allow-any-step-order
-snovault = git https://github.com/ENCODE-DCC/snovault.git rev=1.0.23
+snovault = git https://github.com/ENCODE-DCC/snovault.git branch=SNO-69-backend-matrix
```
Comment (caseylitton) [inline buildout.cfg]: not forget to revert once finished

**Comment by caseylitton** [comment]
closed code is way out of date.

**Pull request closed without merging**